ui/packages/form/src/DropdownOptions.svelte

```diff
@@ -28,7 +29,12 @@
 				data-value={choice}
 				aria-label={choice}
 			>
-				<span class:hide={!value?.includes(choice)} class="inner-item pr-1">
+				<span
+					class:hide={!(Array.isArray(value) ? value : [value])?.includes(
+						choice
+					)}
+					class="inner-item pr-1"
+				>
```

Comment:
This was a bug in single mode. `py` was also matching `python` because `python` conatin `py`

Reply:
Is this the same bug as https://github.com/gradio-app/gradio/issues/3423?

Reply:
yus! will add to body!